lib/index.js

```diff
@@ -1,4 +1,3 @@
 module.exports = {
-    run: require('./run'),
-    version: require('./version')
```

Comment:
Not required because of: https://github.com/postmanlabs/newman/pull/1623/files#diff-024316d2f6b434a9ff0a176917b4ebb3R12